backend/dataall/modules/__init__.py

```diff
@@ -0,0 +1,39 @@
+"""
+Contains all submodules that can be plugged into the main functionality
+
+How to migrate to a new module:
+1) Create your python module
+2) Create the following submodules:
+    a) gql
```

Comment:
curious if we can be less technology specific in naming the submodules, just an idea:
- api
- tasks (are we going to divide this into short vs long running tasks ? )
- infra

Reply:
I was trying to follow the existing approach and named them after the original modules. But I am happy to change them to be more generic. 


`api` - is a great name, but we already have `dataall.api` and `dataall.db.api` packages, so if it doesn't bring confusion to anyone we can use this (as alternative we can use `schema`, but it's more specific) 
`infra` - this one can be confusing from my point of view, since we can do infrastructure changes in lambda and ecs tasks. I was thinking about `ecs` as it's where the code will run (but again platform-specific)
`tasks` - this can be tricky. I thought about two approaches: 
1) To name it tasks and have two submodules `short` and `long` that represent short-living and long-living tasks accordingly (or `lambda` and `ecs` tasks, but it will be specific). This might be more clear for understanding
2) To rename it to `handlers` (as it used to be) and put there only short-living tasks. Long-living tasks will be placed under `ecs/cdk/infra` module. This might make a module importing a little bit easier.

That's very good that you brought this up, now we can decide how to make it correctly!
Naming is hard, anyone is very welcome to add their opinion on how to name  and structure it :)

Reply:
- [x] Renamed gql to api as you suggested. After discussion with @dlpzx, we decide to leave `cdk` and `tasks` for a while. But we might revisit it in the later PRs :)